src/trusted/service_runtime/nacl_syscall_strace.c

```diff
@@ -6,17 +6,36 @@
 #include <stdio.h>
 #include <stdint.h>
 #include <sys/poll.h>
+#include "native_client/src/trusted/service_runtime/include/bits/nacl_syscalls.h"
```

Comment:
Just curious, what is this include for

Reply:
its for the the syscalls defines, this is good